pkg/registry/batch/cronjob/strategy.go

```diff
@@ -114,7 +116,11 @@ func (cronJobStrategy) Validate(ctx context.Context, obj runtime.Object) field.E
 // WarningsOnCreate returns warnings for the creation of the given object.
 func (cronJobStrategy) WarningsOnCreate(ctx context.Context, obj runtime.Object) []string {
 	newCronJob := obj.(*batch.CronJob)
-	return pod.GetWarningsForPodTemplate(ctx, field.NewPath("spec", "jobTemplate", "spec", "template"), &newCronJob.Spec.JobTemplate.Spec.Template, nil)
+	warnings := pod.GetWarningsForPodTemplate(ctx, field.NewPath("spec", "jobTemplate", "spec", "template"), &newCronJob.Spec.JobTemplate.Spec.Template, nil)
+	if strings.Contains(newCronJob.Spec.Schedule, "CRON_TZ") {
+		warnings = append(warnings, fmt.Sprintf("CRON_TZ used in %s is not supported", field.NewPath("spec", "spec", "schedule")))
```

Comment:
not officially supported? known to work inconsistently? people will observe it apparently working and be confused about what this warning means. Should this link to a more complete explanation?

Reply:
Whatever we end up with here, replicate to the update warning

Reply:
I'll be working on a blog post explaining that, we can hold merging this until after that blog is out.

Reply:
Currently we only have sig-apps meeting notes and recording, which is hardly linkable.

Reply:
We can add aliases to blog posts, so something like k8s.io/cron-tz can point there.

Reply:
https://k8s.io/cron-tz-environment ?

Reply:
I've updated with links to https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/ which should soon have https://github.com/kubernetes/website/pull/30509 merged.